documentation/modules/ROOT/pages/_partials/build-containers.adoc

```diff
@@ -42,7 +42,7 @@ After successful maven build, run the following command to build te container:
 [#conk8s-build-quarkus-native-app-container]
 [source,bash,subs="+macros,+attributes"]
 ----
-docker build -f src/main/docker/Dockerfile.native -t example/{quarkus-project-name}-native .
+docker build -f src/main/docker/Dockerfile.native -t example/{quarkus-project-name}:1.0-SNAPSHOT .
```

Comment:
instead of `1.0-SNAPSHOT` shall we can use `1.0-SNAPSHOT-native` that gives a better clarity of what image the users have built